CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-# `dev-master`
+# 4.1.0 - 2022-10-11
```

Comment:
@Grummfy I set date to tomorrow, as I do not know when you will have time to tag/release, but there is no rush.